crates/telio-wg/src/uapi.rs

```diff
@@ -28,6 +28,15 @@ pub enum Error {
     ParsingError(&'static str, String),
 }
 
+#[derive(Debug, Eq, PartialEq, Clone, Copy)]
+/// Possiblie reason of the endpoint change:
+pub enum EndpointChangeReason {
```

Comment:
can we use this enum on [update fn](https://github.com/NordSecurity/libtelio/pull/886/files#diff-e155115f7ca084b5dc3db3e636dd963f8174c17b6d08ec4f7c871df22363f5b0R914) arguments instead of bool?

Reply:
Added this refactor in a separate commit - I guess `UpdateReason` makes more sense when we talk about using it for the update function everywhere. I'm not sure whether it looks better or not tbh, I'm happy with both versions.